### PR TITLE
zh-translation:assign-cpu-resource.md

### PR DESCRIPTION
**Pull request by yuxiaobo96**

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+Container is automatically assigned the default limit. Cluster administrators can use a
+[LimitRange](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core/)
+to specify a default value for the CPU limit.
+-->
+如果您没有为容器指定 CPU 限制，则适用以下情况之一：
+
+* 容器在可以使用的 CPU 资源上没有上限。容器可以使用运行节点上的所有可用的 CPU 资源。
+
+* 容器在具有默认 CPU 限制的命名空间中运行，并且系统会自动为容器分配默认限制。集群管理员可以使用
+[限制范围]（/ docs / reference / generation / kubernetes-api / {{<param“ version”>}} /＃limitrange-v1-core /）
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md]: This is not a valid markdown link.
DO NOT DO SPACE INSERTIONS IN BATCH SCRIPT!

Comment (yuxiaobo96) [reply content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md]: Ok, I will modify it.

File: content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+[LimitRange](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core/)
+to specify a default value for the CPU limit.
+-->
+如果您没有为容器指定 CPU 限制，则适用以下情况之一：
+
+* 容器在可以使用的 CPU 资源上没有上限。容器可以使用运行节点上的所有可用的 CPU 资源。
+
+* 容器在具有默认 CPU 限制的命名空间中运行，并且系统会自动为容器分配默认限制。集群管理员可以使用
+[限制范围](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core/)
+指定CPU限制的默认值。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md]:
```suggestion
指定 CPU 限制的默认值。
```

File: content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+<!--
+By configuring the CPU requests and limits of the Containers that run in your
+cluster, you can make efficient use of the CPU resources available on your cluster
+Nodes. By keeping a Pod CPU request low, you give the Pod a good chance of being
+scheduled. By having a CPU limit that is greater than the CPU request, you accomplish two things:
+
+* The Pod can have bursts of activity where it makes use of CPU resources that happen to be available.
+* The amount of CPU resources a Pod can use during a burst is limited to some reasonable amount.
+-->
+通过配置在集群中运行容器的 CPU 请求和限制，您可以有效地利用在集群节点上的可用CPU资源。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md]:
```suggestion
通过配置在集群中运行容器的 CPU 请求和限制，您可以有效地利用在集群节点上的可用 CPU 资源。
```

File: content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+通过将 Pod CPU 请求保持在较低水平，可以使 Pod 更好的被调度。通过使 CPU 限制大于 CPU 请求，您需要完成以下两件事：
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md]:
```suggestion
通过将 Pod CPU 请求保持在较低水平，可以使 Pod 更好的被调度。通过设置 CPU 限制大于 CPU 请求，您可以实现达到以下两个目的：
```

File: content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md [hunk 5]
```diff
@@ -0,0 +1,490 @@
...
+
+* Pod可能会有大量活动，它利用恰好可用的CPU资源。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md]:
```suggestion
* 在 Pod 上突发大量活动期间，它可以利用节点上碰巧可用的 CPU 资源。
```

File: content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md [hunk 6]
```diff
@@ -0,0 +1,490 @@
...
+* Pod在突发期间可以使用的CPU资源数量被限制为合理的数量。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md]:
```suggestion
* Pod 在突发负载期间可使用的 CPU 资源数量仍被限制为合理的数值。
```

File: content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md [hunk 7]
```diff
@@ -0,0 +1,490 @@
...
+<!-- ### For app developers -->
+### 对于应用程序开发人员
+
+<!--
+* [Assign Memory Resources to Containers and Pods](/docs/tasks/configure-pod-container/assign-memory-resource/)
+
+* [Configure Quality of Service for Pods](/docs/tasks/configure-pod-container/quality-service-pod/)
+-->
+
+* [将内存资源分配给容器和Pod]](/docs/tasks/configure-pod-container/assign-memory-resource/)
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md]:
```suggestion
* [将内存资源分配给容器和 Pod]](/docs/tasks/configure-pod-container/assign-memory-resource/)
```

File: content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md [hunk 8]
```diff
@@ -0,0 +1,490 @@
...
+
+* [配置Pod的服务质量](/docs/tasks/configure-pod-container/quality-service-pod/)
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md]:
```suggestion
* [配置 Pod 的服务质量](/docs/tasks/configure-pod-container/quality-service-pod/)
```

File: content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md [hunk 9]
```diff
@@ -0,0 +1,490 @@
...
+
+* [为命名空间配置最小和最大内存限制](/docs/tasks/administer-cluster/memory-constraint-namespace/)
+
+* [为命名空间配置最小和最大CPU约束](/docs/tasks/administer-cluster/cpu-constraint-namespace/)
+
+* [为命名空间配置内存和CPU配额](/docs/tasks/administer-cluster/quota-memory-cpu-namespace/)
+
+* [为命名空间配置Pod配额](/docs/tasks/administer-cluster/quota-pod-namespace/)
+
+* [配置API对象的配额](/docs/tasks/administer-cluster/quota-api-object/)
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md]: 检查476-488 行的中英文留白。

Comment (yuxiaobo96) [reply content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md]: Thank you !

**Pull request merged**